models/cardSchema.js

```diff
@@ -19,22 +18,8 @@ const CardCategory = [
 
 // Schema
 
-const userSchema = new Schema({
-    email: {
-        type: String,
-        required: [true, 'Email is required'],
-        unique: true
-  }, 
-    password: {
-        type: String,
-        required: [true, 'Password is required']
-    },
-    token: {
-        type: String,
-        default: null
-    },
-  cards: [
-    {
+const cardSchema = new Schema(
+  {
           title: {
             type: String,
             min: 2,
```

Comment:
brakuje jeszcze required dla

- title:
`required: [true, "Set title for card"],`
- date:
`required: [true, "Set date for card"],`
- time:
`required: [true, "Set time for card"],`